airgun/views/report_template.py

```diff
@@ -104,3 +104,4 @@ def is_displayed(self):
             and self.breadcrumb.locations[0] == 'Report Templates'
             and self.breadcrumb.read() == 'Generate a Report'
         )
+        
```

Comment:
From this and the removed `,` in the report_template.py file, it looks like you aren't running pre-commit in your environment. 

Travis is skipping right now, we need to get moved over to a GHA to enforce this.

Reply:
Indeed I wasn't. I've installed the hooks and ran them this time.